frontend/src/components/pages/friends/Friends.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
+import Layout from '../../shared/layout/Layout';
+import UserList from '../../shared/userList/UserList';
+import getFriends from '../../../services/getFriends';
+import './friends.scss';
+
+const Friends: React.FC = () => {
+  const [countFriends, setCountFriends] = useState(0);
+  const friends = getFriends();
```

Comment:
no point to use state for length property...Rather set friends into state and use fetch on useEffect (on mount only)